### PR TITLE
Clear tcmf code

### PR DESCRIPTION
**Pull request by yushan111**
It seems cuda flag is already removed. So code cleaning contains:
* remove deprecate option
* optimize imports
* optimize class and filenames

### CODE REVIEW

**Comment by shane-huang** [comment]
https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/automl/model/tcmf/data_loader.py#L119 according to comment this option is deprecated, if it is deprecated we can just remove it.

**Comment by shane-huang** [comment]
https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/automl/model/tcmf/DeepGLO.py#L408 this mod is useless as mod always >=0. This probably due to tuning. We can remove this condition as it also takes time.

**Comment by shane-huang** [comment]
remove all x=x.cpu

**Comment by yushan111** [comment]
jenkins: http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Validation/3746/

**Comment by yushan111** [comment]
> https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/automl/model/tcmf/DeepGLO.py#L408 https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/automl/model/tcmf/DeepGLO.py#L411 the mod condition at these places are useless as mod always >=0. This probabaly due to intermediate tuning in previous versions. Anyway since this version works well we can just remove this condition as it also takes time.



> remove all x=x.cpu()

Both done

File: pyzoo/zoo/automl/model/tcmf/DeepGLO.py [hunk 1]
```diff
@@ -284,12 +263,12 @@ def predict_future_batch(self, model, inp, future=10):
...
 
         out = self.temporal_to_tensor2d(out)
-        out = np.array(out.cpu().detach())
+        out = np.array(out.detach())
         return out
 
     def predict_future(self, model, inp, future=10, bsize=90):
         n = inp.size(0)
-        inp = inp.cpu()
+        inp = inp
```
Comment (shane-huang) [inline pyzoo/zoo/automl/model/tcmf/DeepGLO.py:271]: why we need this line if we remove .cpu?

File: pyzoo/zoo/automl/model/tcmf/data_loader.py [hunk 2]
```diff
@@ -45,28 +45,11 @@
...
-from scipy.io import loadmat
-
-import torch.nn as nn
-import torch.optim as optim
 import numpy as np
-
-# import matplotlib
-from torch.autograd import Variable
-import itertools
-from sklearn.preprocessing import normalize
```
Comment (shane-huang) [inline pyzoo/zoo/automl/model/tcmf/data_loader.py:60]: we don't need normalize in this file?

Comment (shane-huang) [reply pyzoo/zoo/automl/model/tcmf/data_loader.py:60]: just make sure this lib is not used actually then it is okay for me.

File: pyzoo/zoo/automl/model/tcmf/DeepGLO.py (hunk 1 above)
Comment (shane-huang) [reply pyzoo/zoo/automl/model/tcmf/DeepGLO.py:271]: this line is redundant

**Comment by shane-huang** [comment]
LGTM once above comments are addressed.

File: pyzoo/zoo/automl/model/tcmf/data_loader.py (hunk 2 above)
Comment (yushan111) [reply pyzoo/zoo/automl/model/tcmf/data_loader.py:60]: Normalize is not used in the file. 
Deepglo implements normalization as https://github.com/intel-analytics/analytics-zoo/blob/master/pyzoo/zoo/automl/model/tcmf/DeepGLO.py#L537

File: pyzoo/zoo/automl/model/tcmf/DeepGLO.py (hunk 1 above)
Comment (yushan111) [reply pyzoo/zoo/automl/model/tcmf/DeepGLO.py:271]: will remove it.

**Comment by shane-huang** [comment]
LGTM
